modules/logging/src/main/java/io/oasp/module/logging/common/impl/SecureLogging.java

```diff
@@ -0,0 +1,242 @@
+package io.oasp.module.logging.common.impl;
+
+import java.lang.reflect.Method;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Marker;
+import org.slf4j.MarkerFactory;
+
+/**
+ * Class which provides {@link Marker}s for differential logging. Implements "MultiMarker"s for optimal filtering (if
+ * dependency org.owasp available), or corresponding conventional Markers as a fall back solution.
+ * <P>
+ * Example usage:
+ *
+ * <pre>
+ * {@code
+ * LOG.info(SecureLogging.SECURITY_FAILURE_CONFIDENTIAL, "Confidential Security Failure message.");
```

Comment:
Nice to have but I would propose to use {@link SecureLogging},{@link #SECURITY_FAILURE_CONFIDENTIAL} etc.
This makes the JavaDoc even more specific and expressive, ensures there are no typos and is refactoring safe.